defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
-# defaults file for dconf-settings
\ No newline at end of file
+# defaults file for dconf-settings# defaults file for dconf-settings
```

Comment:
Looks like you accidentally duplicated this

Reply:
fixed